VideoRenderer/VideoRenderer/RendererRepository.swift

```diff
@@ -124,7 +124,7 @@ extension Renderer {
 
 extension Renderer {
     public struct Descriptor {
-        /// Example: com.aol.onemobilesdk.flat
+        /// Example: com.aol.VerizonVideoPartnerSDK.flat
```

Comment:
I don't see the changes in renderers where you've changed the descriptor, it is still `com.onemobilesdk.videorenderer.flat`
Maybe we go with something like "com.Verizon.VideoPartnerSDK.VideoRenderer.flat' or whatever?

Reply:
It is just comment change - we will not change it now - all identifiers will remain the same - because it will require M/S changes.

Reply:
Alright then.